xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/edit.less

```diff
@@ -2,34 +2,51 @@
 // Edit
 // --------------------------------------------------
 
-
-.bottombuttons {
+.sticky-buttons {
```

Comment:
replacing this class might have some problems in the places that were using it.

Reply:
This css rule wasn't used before.

Reply:
Most of the changed have been introduced by #738.
This code makes sure to keep the previous behavior outside the `.sticky-buttons` element:

```css
#editcolumn, #inline {
  :not(.sticky-buttons).bottombuttons {
    .buttons {
      margin: 10px 0;
    }
  }
...
```